### ISSUE TITLE
Model initialization from scratch

### ISSUE THREAD

**Issue by aleksandrmelnyk** [MarvinTeichmann/tensorflow-fcn#34]
Can the fcn model be initialized without vgg16.npy?  Why it requires the saved model?

**Comment by MarvinTeichmann** [comment]
Yes, change the initializer to a random one, e.g [here](https://github.com/MarvinTeichmann/tensorflow-fcn/blob/master/fcn8_vgg.py#L274).

**Comment by theladyjaye** [comment]
@aleksandrmelnyk  did you have success with this?
